bin/ci-studio-common-util

```diff
@@ -34,7 +34,22 @@ function allow_user() {
   chmod 440 "/etc/sudoers.d/$user"
 }
 
+function lock() {
+  exec 200>/var/lock/ci-studio-common-update.lock
+
+  flock -n 200 \
```

Comment:
[nit/question] Isn't flock's exit code is already going to be non-zero on failure, zero on success? Oh, is this to make it work with set -e?